### PR TITLE
Fixed library support for Python 3

### PR DESCRIPTION
**Pull request by holgerd77**
Hi, just added the suggested Python 3 compatibility fix from @jschilling1 suggested here https://github.com/scrapy-plugins/scrapy-djangoitem/issues/10

I tested this with Python 3.5 and Python 2.7.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,5 +26,5 @@
         'Framework :: Django',
         'Framework :: Scrapy',
     ],
-    requires=['scrapy (>=0.24.5)', 'django'],
+    requires=['scrapy (>=0.24.5)', 'django', 'six'],
```
Comment (kmike) [inline setup.py]: Could you please move `six` from requires to install_requires?

Comment (kmike) [inline setup.py]: It also makes sense to add Python 3 versions to classifiers

**Comment by kmike** [comment]
:+1: awesome, thanks @holgerd77!

**Comment by holgerd77** [comment]
Done.

**Pull request merged**